sogs/omq.py

```diff
@@ -0,0 +1,63 @@
+# Common oxenmq object; this is used by workers and the oxenmq mule.  We create, but do not start,
+# this pre-forking.
+
+import oxenmq
+from oxenc import bt_serialize
+
+from . import crypto, config
+from .postfork import postfork
+
+omq = None
+mule_conn = None
+
+
+def make_omq():
+    omq = oxenmq.OxenMQ(privkey=crypto._privkey.encode(), pubkey=crypto.server_pubkey.encode())
```

Comment:
the public keys will be the server's pubkey when they come from the http handlers, is that desired?

Reply:
For workers it doesn't matter (they aren't using a curve connection anyway); for the mule, yes, this is desired.